src/components/Modal/index.js

```diff
@@ -28,25 +28,23 @@ export default function Modal({
     return pushData;
   };
 
-  const [isRoundModal, setIsRoundModal] = useState(isInterim);
-
   return (
     <>
       {!isfinishResult ? (
         <>
-          <ModalDiv onClick={handleClick}>
-            <p>{isRoundModal ? "중간 결과" : "라운드 결과"}</p>
-            {isRoundModal ? (
+          <ModalDiv onClick={handleToggleBoard}>
+            <p>{isInterim ? "중간 결과" : "라운드 결과"}</p>
```

Comment:
변수명을 보고 이해가 안갑니다 . isInterim이 뭘까요? 뭔가 확 와닿지가 않은데, 혹시... 중간 5라 마치고 모달이 갑자기 뜨는 거 확인용일까요?
```suggestion
            <p>{isInterim ? "중간 결과" : "라운드 결과"}</p>
```

Reply:
중간결과를 표시할때 사용하는 변수인데, 적절한 네이밍으로 교체 해보겠습니다!